Sources/PaymentsCore/Models/OrderData.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public struct OrderData {
```

Comment:
This is something Android can align on. Having a more generic `OrderData` class does make sense. On the other hand, there is some additional information from the `confirm-payment-source` endpoint response like `lastDigits` and `cardBrand` do we feel like the merchant will need this info?

Reply:
I think after some discussions with David we landed on a parent class living in Core, and then subclass result types living in each payment module. `CardResult`, `PayPalResult` etch to contain the more specific info.

Not sure about `OrderData` naming